src/lib.rs

```diff
@@ -96,6 +96,18 @@ impl<K, V> LruCache<K, V> where K: PartialOrd + Ord + Clone {
             self.map.insert(key, (value, time::SteadyTime::now()));
         }
     }
+/// Remove a key/value pair from cache
```

Comment:
Is there a reason to remove this Brian, I think Mahmoud may use this in vaults somewhere, at least he added a remove to lru_Cache for a reason I cannot quite remember just now.